services/core/shared/core/compat/common/wsRequest.js

```diff
@@ -42,6 +44,11 @@ const instantiateClient = async () => {
 			}
 			return clientCache;
 		}
+
+		if ((Date.now() - instantiationBeginTime) > 500) {
```

Comment:
Can you please give that `500` a name?
As previously discussed, did you consider a different value?

Reply:
Done!
The client instantiation time varies between `9-20ms`. So, updated the wait time to be `50ms`.